drivers/gles3/rasterizer_storage_gles3.h

```diff
@@ -1394,6 +1394,7 @@ class RasterizerStorageGLES3 : public RasterizerStorage {
 	void _render_target_allocate(RenderTarget *rt);
 
 	virtual RID render_target_create();
+	virtual RID render_target_screen_create(const Rect2 &p_screen_rect);
```

Comment:
Just make it a flag instead of a new function, like RENDER_TARGET_DIRECT_TO_SCREEN or something like this.

Reply:
That would be a flag that is passed into render_target_create()?